docs/usage/index.md

```diff
@@ -79,29 +79,85 @@ To avoid this situation, one solution is to run {ref}`Pyodide in a WebWorker <us
 
 ## Node.js
 
-As of version 0.18.0 Pyodide can experimentally run in Node.js.
+```{note}
+The following instructions have been tested with Node.js 18.5.0. To use
+Pyodide with older versions of node, you might need special command line
+instructions, see below.
+```
 
-Install the [Pyodide npm package](https://www.npmjs.com/package/pyodide),
+It is now possible to install the
+[Pyodide npm package](https://www.npmjs.com/package/pyodide) in Node.js. To
+follow these instructions you need at least pyodide 0.21.0, which at the
+moment of writing has not been released yet. You can explicity ask npm to use
```

Comment:
```suggestion
follow these instructions you need at least pyodide 0.21.0.
You can explicitly ask npm to use
```
 
Since the docs get published to the dev docs https://pyodide.org/en/latest/ and by the time it gets deployed to the stable docs which users use https://pyodide.org/en/stable, the release would happen.